### DISCUSSION TITLE
Is there a way to mark particular discussions as urgent?

### DISCUSSION THREAD

**Discussion by joel-ekstep** [community/community#3292] [Discussions] [answered]
We are building an open source project. Some of our remote team members start new discussions, and require a quick response. However, since there are many ongoing discussions, the new ones tend to get lost in the crowd. 

The workaround right now is to have a daily email thread that specifies the critical discussions that need resolution that day - but this completely destroys the benefit of moving from email to open discussions, if email is still being used as a support system.

Is there a way to mark specific discussions as top priority, or urgent?

**Comment by joel-ekstep** [comment]
Dropping a reminder on this, in case anyone knows a good way to resolve this.

**Comment by PranamBhat** [comment, marked as answer]
There is no such feature on GitHub. You can always tag someone (GitHub users) and also raise your query over GitHub "Feedback" section.

**Reply by joel-ekstep** [reply]
Have recently started using labels for this, with a bright red color. Here's hoping it works.
